airflow/lineage/__init__.py

```diff
@@ -16,10 +16,16 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
+#
+"""Define the objects `prepare_lineage` and `apply_lineage`
```

Comment:
I think would be great to move the """ to separate line as mentioned by @feluelle

Reply:
thank you Jarek, I apply it 👍